charts/headlamp/values.yaml

```diff
@@ -92,6 +92,8 @@ config:
   # -- directory to look for plugins
   pluginsDir: "/headlamp/plugins"
   watchPlugins: false
+  tlsCert: "/headlamp-cert/headlamp-ca.crt"
```

Comment:
Does having these set by default mean the files are required?

I don't think we'd want to require these files (or volume mounts) by default, especially if it breaks existing users.